lib/morpheus.gi

```diff
@@ -18,34 +18,101 @@ MORPHEUSELMS := 50000;
 
 InstallMethod(Order,"for automorphisms",true,[IsGroupHomomorphism],0,
 function(hom)
-local map,phi,o,lo,i,start,img;
+local map,phi,o,lo,i,j,start,img,d,nat,ser,jord,first;
+  d:=Source(hom);
```

Comment:
A short comment here as to what this method does would be very helpful. Right now, I see the commit message, which heelps, but if I read that code in a year, I won't know about it... BTW the commit message says a "characteristic series" is computed, but in some cases it only checks if the series is invariant under `hom` (which of course is enough)

So, wow about this
> `#` To compute the order, use a suitable characteristic series 'ser' and work in the factor groups.

(And perhaps "characteristic series" should be "series invariant under 'hom'" ?

Reply:
is in now